docker/fedora-25/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM        fedora:25
 MAINTAINER  Anton Povarov "anton.povarov@gmail.com"
 
+# TODO: Use multi-stage builds https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
```

Comment:
Using multi-stage `Dockerfile` will allow removing cleanup which does not work as expected because of nature of layers :
```Dockerfile
 RUN         dnf remove -y git-core gcc gcc-c++ boost-devel cmake autoconf automake libtool mariadb-devel && \
 			dnf autoremove -y && \
 			dnf clean all
```